src/Mono.Android/map.csv

```diff
@@ -5195,3 +5221,19 @@
 24,Java.Util.SpliteratorCharacteristics,Sized,I:java/util/Spliterator.SIZED,64
 24,Java.Util.SpliteratorCharacteristics,Sorted,I:java/util/Spliterator.SORTED,4
 24,Java.Util.SpliteratorCharacteristics,Subsized,I:java/util/Spliterator.SUBSIZED,16384
+0,Android.Content.PM.LauncherAppsShortcutQueryFlags,None,,0
```

Comment:
This has been historically done for any "extra default" enum values to API Level 0 (since enum values can be added at any API level, there is no such a mapping that "this enum type is for API Level X" (unless we collect the lowest minimum API Level at generator). And if we make such a change, that will break existing enums API. And I don't see any point of special-casing this specific enum type. Lastly, it is harmless.